ansible/templates/usb/scripts/prompt-config.sh.j2

```diff
@@ -2045,6 +2045,18 @@ web management.\n\n"
 	[[ -z "${cfgdb_admin_ip}" ]] && cfgdb_admin_ip="$(discover_cfgdb)"
 	prompt_host_ok_val "Configuration database IP address" "${cfgdb_admin_ip}" "cfgdb_admin_ip"
 	cfgdb_admin_ip="${val}"
+	if [[ ${advanced_install} -eq 1 ]] && ! ip_from_subnet "${cfgdb_admin_ip}" "${admin_ip}" "${admin_netmask}" ; then
+		# the CFGDB IP is outside the subnet
+		echo
+		echo "You have entered an IP address outside the admin subnet."
+		echo "Remote node connects to the central management"
```

Comment:
node -> compute node
management -> management VM
admin subnet -> admin network

(so we have at least some consistency with already used words/terms)